src/status/libstatus/stickers.nim

```diff
@@ -0,0 +1,50 @@
+import eth/common/eth_types
+import ./core as status, ./types, ../contracts, ./settings, ./utils
```

Comment:
libstatus files should not import from `../` (just like, say, status-go is it's own thing and shouldn't/can't import from status-react)